lib/messages/status.js

```diff
@@ -4,9 +4,13 @@ const {
 } = require('.');
 
 class Status extends Message {
-  constructor(status) {
+  constructor(checkOrStatus) {
     super({});
-    this.status = status;
+    this.status = checkOrStatus.state || checkOrStatus.conclusion || checkOrStatus.status;
```

Comment:
The `checkOrStatus.conclusion` thing is confusing here but I get what you're doing because for check runs the most important thing is the `conclusion` (e.g. `success`). It was confusing because you're storing this in a `this.status` variable. Maybe we could use a better name for that variable, but in any case looks good to me 👍

Reply:
Agreed about it being confusing! How's `combinedStatus` or `state`?